src/framework/components/rigid-body/system.js

```diff
@@ -69,8 +75,20 @@ class RaycastResult {
 }
 
 /**
- * Object holding the result of a contact between two rigid bodies.
+ * Represents the detailed data of a single contact point between two rigid bodies in the physics
+ * simulation. This class provides comprehensive information about the contact, including the
+ * entities involved, the exact contact points in both local and world space coordinates, the
+ * contact normal, and the collision impulse force.
+ *
+ * Instances of this class are created by the physics engine when collision events occur and are
+ * passed to event handlers only through the global `contact` event on the
+ * {@link RigidBodyComponentSystem}. Individual rigid body components receive instances of
+ * {@link ContactResult} instead.
  *
+ * @example
+ * app.systems.rigidbody.on('contact', (result) => {
+ *     console.log(`Contact between ${result.a.name} and ${result.b.name}`);
```

Comment:
There are no a and b properties in a contact result.

Reply:
This is a SingleContactResult. 🙂